vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
The benefit of named parameters is documentation, just from looking at the function type I can't infer what the parameters are and you didn't add documentation. You can indeed do:
```
val notation_activation :
?notation:Constrexpr.notation -> ?scope:Notation_term.scope_name -> bool -> uni
```
and now it becomes clear that we have no clue what the meaning for `notation_activation true` is.